library/include/MiscUtils.h

```diff
@@ -389,6 +386,30 @@ DFHACK_EXPORT std::string toUpper(const std::string &str);
 DFHACK_EXPORT std::string toLower(const std::string &str);
 DFHACK_EXPORT std::string to_search_normalized(const std::string &str);
 
+static inline std::string int_to_string(const int n)
+{
+    std::ostringstream ss;
+    ss << n;
+    return ss.str();
+}
+
+// trim from start
+static inline std::string &ltrim(std::string &s) {
+    s.erase(s.begin(), std::find_if(s.begin(), s.end(), [](char x){ return !std::isspace(x); }));
```

Comment:
On Windows (Buildmaster):
```
z:\home\buildmaster\work\_e0\src\library\include\MiscUtils.h(398): error C2039: 'isspace': is not a member of 'std'
```

not sure where it's sneaking in on Linux, but you probably want to `#include <cctype>`

Reply:
Done. Thanks!